usr/bin/playtronos-systemreport

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# System Report
+#
+# Generates a system report archive which can contain useful information for
+# reporting bugs and troubleshooting.
+
+if [ $EUID -ne 0 ]; then
+  echo "$(basename "$0") must be run as root"
+  exit 1
+fi
+
+ENDCOLOR='\e[0m'
+RED='\e[0;31m'
+PURPLE='\e[0;35m'
+CYAN='\e[0;36m'
+
+TARGET_USER=playtron
+REPORT_NAME="report-$(date +%Y-%m-%d-%s)"
+REPORT_DIR_BASE="$(mktemp -d)"
+REPORT_DIR="${REPORT_DIR_BASE}/${REPORT_NAME}"
+mkdir -p "${REPORT_DIR}"
+
+echo -e "${CYAN}Generating system report...${ENDCOLOR}"
+
+# Hardware
+echo -e "  Collecting ${RED}hardware${ENDCOLOR} report..."
+dmesg >"${REPORT_DIR}/dmesg.log"
+lspci -tv -nn >"${REPORT_DIR}/lspci.out"
+lsusb -t -v >"${REPORT_DIR}/lsusb.out"
+cat /proc/bus/input/devices >"${REPORT_DIR}/input-devices.out"
+hwctl system-info >"${REPORT_DIR}/system.json" 2>/dev/null
+
+# Firmware
+fwupdmgr get-devices >"${REPORT_DIR}/firmware-devices.out" 2>&1
+fwupdmgr get-updates -y >"${REPORT_DIR}/firmware-updates.out" 2>&1
+
+# Processes
+echo -e "  Collecting ${RED}process${ENDCOLOR} report..."
+ps aux >"${REPORT_DIR}/processes.out"
```

Comment:
Can we add `-f` to also follow forked processes? `ps afux`

For example,

before:

```
root        1477  0.0  0.1 156512 22592 ?        Ssl  17:07   0:00 /usr/bin/sddm
```

after:

```
root        1477  0.0  0.1 156512 22592 ?        Ssl  17:07   0:00 /usr/bin/sddm
root        1479  0.0  0.1  74284 20680 ?        S    17:07   0:00  \_ /usr/libexec/sddm-helper --socket /tmp/sddm-auth-0d12d4ec-f30f-4f13-8c73-12c1f09b6681 --id 1 --start gamescope-session-plus playtron --user playtron --autologin
playtron    1508  0.0  0.0   7076  3564 tty1     Ss+  17:07   0:00      \_ /usr/bin/sh /usr/bin/gamescope-session-plus playtron
playtron    1590  0.0  0.0  18868  7500 tty1     S+   17:07   0:00          \_ systemctl --user --wait start gamescope-session-plus@playtron.service
```